frontend/src/AlertConfiguration/AlertConfigurationComponents/ControlPanel.js

```diff
@@ -1,49 +1,65 @@
-import React, { useState, Fragment } from 'react';
-import AppBar from '@material-ui/core/AppBar';
-import CssBaseline from '@material-ui/core/CssBaseline';
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
 import { makeStyles } from '@material-ui/core/styles';
-import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
-import ControlPanel from './ControlPanel';
-import Footer from './Footer';
-import ConfigList from './ConfigList';
+import ConfigForm from './ConfigForm'
 
-const useStyles = makeStyles((theme) => ({
-  main: {
-    display: 'flex',
-    flexDirection: 'column',
-    minHeight: '100vh',
+const useStyles = makeStyles({
+  controlPanel: {
+    backgroundColor: 'white',
+    padding: 50,
+    width: '100wv'
   },
-}));
+  configForm: {
+    padding: 25,
+  },
+});
 
-export default function AlertConfiguration() {
+export default function ControlPanel({ addConfig }) {
   const classes = useStyles();
 
-  const [configs, setConfigs] = useState([]);
-
-  function addConfig(config) {
-    setConfigs([config, ...configs]);
-  }
-
-  return (
-    <Fragment>
-      <CssBaseline />
-      {/* TODO: Replace with Catherine's NavBar */}
-      {/* Navigation bar START */}
-      <AppBar position="relative">
-        <Toolbar>
-          <Typography variant="h6" color="inherit" noWrap>
-            Alert Configuration
-          </Typography>
-        </Toolbar>
-      </AppBar>
-      {/* Navigation bar END */}
+  const [configForm, setConfigForm] = React.useState(false);
+  
+  const handleConfigForm = () => {
```

Comment:
from my understanding, this will toggle the config form to show up / not show-up is that correct? should we rename the `configForm` variable to make it clear its the displaying?

Reply:
Could you please elaborate on what it is you mean?

Reply:
I think Paula means that you should rename the variable `configForm` to something like `showConfigForm` or `displayConfigForm` to make it clear that the state represents whether the form is being shown or not

Reply:
for clarification: I have the same comment about changing the variable name, and suggest changing it to `showConfigForm` or `displayConfigForm` but feel free to change it to whatever you think makes it most clear!

Reply:
Completed